### DISCUSSION TITLE
Github desktop push as github user

### DISCUSSION THREAD

**Discussion by apepenkov** [community/community#24088] [Apps, API and Webhooks]
<p>Hello. I have a question. Is is possible to push updates as GitHub user in github desktop?<br>
<img src="https://user-images.githubusercontent.com/39992738/181081878-844baaf0-e913-471c-b6a2-2c27dacc79ec.png" alt="изображение" data-base62-sha1="3t2FCTLZ0HoAjnVC7P7Xzi98myO" width="276" height="42"><br>
Currently it pushes like this (no account)</p>

**Comment by steveward** [comment, marked as answer]
<p>Welcome <a class="mention" href="/apepenkov">@apepenkov</a>! GitHub Desktop uses the email address in your <code>.gitconfig</code> to associate your commits with the correct GitHub account. You can check this email address by going to <code>File</code> &gt; <code>Options</code> &gt; <code>Git</code> in GitHub Desktop. Let me know if you have any other questions about this.</p>
